packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/mcp/formatters.py

```
@@ -220,6 +287,204 @@ def format_conflict_analysis(conflicts: dict[str, Any]) -> str:
+    ) -> dict[str, Any]:
+        """Create lightweight conflict results for fast navigation and lazy loading."""
+        
+        conflicting_pairs = conflicts.get("conflicting_pairs", [])
+        
+        # Create conflict index with comprehensive information
+        conflict_index = []
+        involved_document_ids = set()
+        
+        for i, (doc1_id, doc2_id, conflict_info) in enumerate(conflicting_pairs):
+            conflict_id = f"conf_{i+1:03d}"
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)